src/terminal/input/mouseInput.cpp

```diff
@@ -438,14 +438,14 @@ std::wstring TerminalInput::_GenerateDefaultSequence(const COORD position,
     //   stream without bash.exe trying to convert it into utf8, and generating extra bytes in the process.
     if (position.X <= s_MaxDefaultCoordinate && position.Y <= s_MaxDefaultCoordinate)
     {
-        const COORD vtCoords = _winToVTCoord(position);
-        const short encodedX = _encodeDefaultCoordinate(vtCoords.X);
-        const short encodedY = _encodeDefaultCoordinate(vtCoords.Y);
+        const auto vtCoords = _winToVTCoord(position);
+        const auto encodedX = _encodeDefaultCoordinate(vtCoords.X);
+        const auto encodedY = _encodeDefaultCoordinate(vtCoords.Y);

```

Comment:
These 3 functions return int instead of short now, so we need to change the code below to use narrow casts.